FreeRTOS/FreeRTOS-Plus-TCP
# Fix warnings on IAR

Description
-----------
This PR fixes the build errors and compilation warnings for the IAR compiler. 

Test Steps
-----------
Build succeeds without errors or warnings.

Checklist:
----------

Related Issue
-----------
Build issues and warnings in IAR build.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
